build_library/prod_image_util.sh

```diff
@@ -100,6 +101,8 @@ create_prod_image() {
           || die_notrace "coreos-au-key is missing the 'official' use flag"
   fi
 
+  tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
+
```

Comment:
Eventually, we should also capture `"${BUILD_DIR}"/configroot` and pass it as `PORTAGE_CONFIGROOT` later when generating sysexts, but I guess the above works for now.

I've created a follow-up issue https://github.com/flatcar/Flatcar/issues/1052 to keep track of this.

Reply:
I think configroot is considered as deprecated and should always be the same as sysroot. Not sure if it's relevant, though, See https://dev.gentoo.org/~zmedico/portage/doc/man/emerge.1.html (search for PORTAGE_CONFIGROOT there).

Reply:
That's a good point (and also, TIL! thanks for sharing) but it's not quite what I meant. *Currently*, `build_image` uses a config root external to the OS image rootfs, via `prod_image_util.sh::create_prod_image()` -> `build_image_util.sh::emerge_to_image()`, here: https://github.com/flatcar/scripts/blob/main/build_library/build_image_util.sh#L183

All I'm saying is that at some point (not necessarily in your PR though) we should capture the contents from there, too, to accurately recreate the base OS image including all portage metadata when we install sysext packages on top.

But doing this cleanly is for https://github.com/flatcar/Flatcar/issues/1052 to look into. No reason to feature-creep this into your work here, Krzesimir.